components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/FontResources.kt

```diff
@@ -35,6 +35,15 @@ expect fun Font(
     style: FontStyle = FontStyle.Normal
 ): Font
 
+
+@Composable
+expect fun Font(
```

Comment:
1) for any public API we need a documantation
2) do we need a function with all parameters? (variation and standard)